Index: b/python/pykde4/CMakeLists.txt
===================================================================
--- a/python/pykde4/CMakeLists.txt
+++ b/python/pykde4/CMakeLists.txt
@@ -31,8 +31,8 @@
 MACRO_OPTIONAL_FIND_PACKAGE(Soprano)
 MACRO_LOG_FEATURE(Soprano_FOUND "Soprano" "Soprano Libraries" "kdesupport" FALSE "" "Required for Soprano Python bindings.")
 
-MACRO_OPTIONAL_FIND_PACKAGE(Nepomuk)
-MACRO_LOG_FEATURE(Nepomuk_FOUND "Nepomuk" "Nepomuk Libraries" "kdesupport" FALSE "" "Required for Nepomuk Python bindings.")
+#MACRO_OPTIONAL_FIND_PACKAGE(Nepomuk)
+#MACRO_LOG_FEATURE(Nepomuk_FOUND "Nepomuk" "Nepomuk Libraries" "kdesupport" FALSE "" "Required for Nepomuk Python bindings.")
 
 MACRO_OPTIONAL_FIND_PACKAGE(KdepimLibs)
 MACRO_LOG_FEATURE(KDEPIMLIBS_FOUND "Kdepimlibs" "KDE pim Libraries" "kdepimlibs" FALSE "" "Required for Akonadi Python bindings.")
@@ -90,10 +90,10 @@
     SET(PYKDE_MODULES "${PYKDE_MODULES} soprano")
 ENDIF(Soprano_FOUND)
 
-IF(Nepomuk_FOUND)
-    ADD_SIP_PYTHON_MODULE(PyKDE4.nepomuk sip/nepomuk/nepomukmod.sip ${NEPOMUK_LIBRARIES} ${QT_QTCORE_LIBRARY}  ${SOPRANO_LIBRARIES})
-    SET(PYKDE_MODULES "${PYKDE_MODULES} nepomuk")
-ENDIF(Nepomuk_FOUND)
+#IF(Nepomuk_FOUND)
+#    ADD_SIP_PYTHON_MODULE(PyKDE4.nepomuk sip/nepomuk/nepomukmod.sip ${NEPOMUK_LIBRARIES} ${QT_QTCORE_LIBRARY}  ${SOPRANO_LIBRARIES})
+#    SET(PYKDE_MODULES "${PYKDE_MODULES} nepomuk")
+#ENDIF(Nepomuk_FOUND)
 
 IF(KDEPIMLIBS_FOUND)
 	ADD_SIP_PYTHON_MODULE(PyKDE4.akonadi sip/akonadi/akonadimod.sip ${KDE4_AKONADI_LIBS} ${KDE4_AKONADI_KMIME_LIBS} ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${KDE4_KDECORE_LIBS} ${KDE4_KDEUI_LIBS})
