diff -urNad libqt4-ruby-1.4.9~/ruby/qtruby/bin/rbqtapi libqt4-ruby-1.4.9/ruby/qtruby/bin/rbqtapi
--- libqt4-ruby-1.4.9~/ruby/qtruby/bin/rbqtapi	2007-04-20 15:03:22.000000000 +0200
+++ libqt4-ruby-1.4.9/ruby/qtruby/bin/rbqtapi	2008-04-29 20:37:31.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/env ruby1.8
 
 # Note: this program is part of qtruby and makes use of its internal functions.
 #       You should not rely on those in your own programs.
