Index: b/ruby/CMakeLists.txt
===================================================================
--- a/ruby/CMakeLists.txt
+++ b/ruby/CMakeLists.txt
@@ -104,8 +104,8 @@
         set(KTEXTEDITOR_ENABLED "yes")
     endif(ENABLE_KTEXTEDITOR_RUBY)
     if(Nepomuk_FOUND)
-	    #add_subdirectory( nepomuk )
-         #set(NEPOMUK_ENABLED "yes")
+        add_subdirectory( nepomuk )
+        set(NEPOMUK_ENABLED "yes")
     endif(Nepomuk_FOUND)
     if(OKULAR_FOUND)
         add_subdirectory( okular )
