Kimono external library should be imported as libkimono

--- a/csharp/kimono/kde/KUrlExtras.cs
+++ b/csharp/kimono/kde/KUrlExtras.cs
@@ -13,13 +13,13 @@
 		public class List : List<KUrl> {
 			delegate bool GetNextDictionaryEntryFn(ref IntPtr key, ref IntPtr value);
 			
-			[DllImport("kimono", CharSet=CharSet.Ansi)]
+			[DllImport("libkimono", CharSet=CharSet.Ansi)]
 			static extern void KUrlListPopulateMimeData(SmokeMarshallers.NoArgs getNextItem, IntPtr mimeData, GetNextDictionaryEntryFn getNextDictionaryEntry, uint flags);
-			[DllImport("kimono", CharSet=CharSet.Ansi)]
+			[DllImport("libkimono", CharSet=CharSet.Ansi)]
 			static extern void KUrlListMimeDataTypes(SmokeMarshallers.FromIntPtr fn);
-			[DllImport("kimono", CharSet=CharSet.Ansi)]
+			[DllImport("libkimono", CharSet=CharSet.Ansi)]
 			static extern bool KUrlListCanDecode(IntPtr mimeData);
-			[DllImport("kimono", CharSet=CharSet.Ansi)]
+			[DllImport("libkimono", CharSet=CharSet.Ansi)]
 			static extern void KUrlListFromMimeData(SmokeMarshallers.FromIntPtr addfn, IntPtr mimeData, GetNextDictionaryEntryFn getNextDictionaryEntry);
 			
 			public List() {}
