diff -Naur kdebindings-4.3.3.orig/csharp/kimono/examples/kio/kio_monodoc/CMakeLists.txt kdebindings-4.3.3/csharp/kimono/examples/kio/kio_monodoc/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/kimono/examples/kio/kio_monodoc/CMakeLists.txt	2008-10-30 15:10:18.000000000 +0100
+++ kdebindings-4.3.3/csharp/kimono/examples/kio/kio_monodoc/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -14,6 +14,7 @@
 
 	set(CS_FLAGS -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll,${LIBRARY_OUTPUT_PATH}/kde-dotnet.dll ${MONODOC_LDFLAGS})
 	add_cs_library(kio_monodoc "${kio_monodoc_src}" ALL)
+	ADD_DEPENDENCIES(kio_monodoc kde-dotnet)
 
 	install(FILES ${LIBRARY_OUTPUT_PATH}/kio_monodoc.dll DESTINATION ${DATA_INSTALL_DIR}/kio_monodoc RENAME main.dll)
 	install(FILES monodoc.protocol DESTINATION ${SERVICES_INSTALL_DIR})
diff -Naur kdebindings-4.3.3.orig/csharp/kimono/tutorials/p1/CMakeLists.txt kdebindings-4.3.3/csharp/kimono/tutorials/p1/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/kimono/tutorials/p1/CMakeLists.txt	2009-01-16 16:04:13.000000000 +0100
+++ kdebindings-4.3.3/csharp/kimono/tutorials/p1/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_P1 ${CMAKE_CURRENT_SOURCE_DIR}/p1.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(p1 ${SRC_P1} ALL)
+ADD_DEPENDENCIES(p1 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/kimono/tutorials/p2/CMakeLists.txt kdebindings-4.3.3/csharp/kimono/tutorials/p2/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/kimono/tutorials/p2/CMakeLists.txt	2009-01-16 16:04:13.000000000 +0100
+++ kdebindings-4.3.3/csharp/kimono/tutorials/p2/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_P2 ${CMAKE_CURRENT_SOURCE_DIR}/p2.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/kde-dotnet.dll)
 ADD_CS_EXECUTABLE(p2 ${SRC_P2} ALL)
+ADD_DEPENDENCIES(p2 kde-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/kimono/tutorials/p3/CMakeLists.txt kdebindings-4.3.3/csharp/kimono/tutorials/p3/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/kimono/tutorials/p3/CMakeLists.txt	2009-01-16 16:04:15.000000000 +0100
+++ kdebindings-4.3.3/csharp/kimono/tutorials/p3/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_P3 ${CMAKE_CURRENT_SOURCE_DIR}/p3.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/kde-dotnet.dll)
 ADD_CS_EXECUTABLE(p3 ${SRC_P3} ALL)
+ADD_DEPENDENCIES(p3 kde-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/kimono/tutorials/p5/CMakeLists.txt kdebindings-4.3.3/csharp/kimono/tutorials/p5/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/kimono/tutorials/p5/CMakeLists.txt	2009-01-16 16:04:16.000000000 +0100
+++ kdebindings-4.3.3/csharp/kimono/tutorials/p5/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_P5 ${CMAKE_CURRENT_SOURCE_DIR}/p5.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/kde-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/khtml-dll.dll)
 ADD_CS_EXECUTABLE(p5 ${SRC_P5} ALL)
+ADD_DEPENDENCIES(p5 khtml-dll)
diff -Naur kdebindings-4.3.3.orig/csharp/kimono/tutorials/p6/CMakeLists.txt kdebindings-4.3.3/csharp/kimono/tutorials/p6/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/kimono/tutorials/p6/CMakeLists.txt	2009-01-16 16:04:16.000000000 +0100
+++ kdebindings-4.3.3/csharp/kimono/tutorials/p6/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_P6 ${CMAKE_CURRENT_SOURCE_DIR}/p6.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/kde-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/khtml-dll.dll)
 ADD_CS_EXECUTABLE(p6 ${SRC_P6} ALL)
+ADD_DEPENDENCIES(p6 khtml-dll)
diff -Naur kdebindings-4.3.3.orig/csharp/kimono/tutorials/p7/CMakeLists.txt kdebindings-4.3.3/csharp/kimono/tutorials/p7/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/kimono/tutorials/p7/CMakeLists.txt	2009-01-16 16:04:16.000000000 +0100
+++ kdebindings-4.3.3/csharp/kimono/tutorials/p7/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_P7 ${CMAKE_CURRENT_SOURCE_DIR}/p7.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/kde-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/khtml-dll.dll)
 ADD_CS_EXECUTABLE(p7 ${SRC_P7} ALL)
+ADD_DEPENDENCIES(p7 khtml-dll)
diff -Naur kdebindings-4.3.3.orig/csharp/kimono/tutorials/p8/CMakeLists.txt kdebindings-4.3.3/csharp/kimono/tutorials/p8/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/kimono/tutorials/p8/CMakeLists.txt	2009-01-16 16:04:16.000000000 +0100
+++ kdebindings-4.3.3/csharp/kimono/tutorials/p8/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_P8 ${CMAKE_CURRENT_SOURCE_DIR}/p8.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/kde-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/khtml-dll.dll)
 ADD_CS_EXECUTABLE(p8 ${SRC_P8} ALL)
+ADD_DEPENDENCIES(p8 khtml-dll)
diff -Naur kdebindings-4.3.3.orig/csharp/kimono/tutorials/p9/CMakeLists.txt kdebindings-4.3.3/csharp/kimono/tutorials/p9/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/kimono/tutorials/p9/CMakeLists.txt	2008-11-19 11:17:31.000000000 +0100
+++ kdebindings-4.3.3/csharp/kimono/tutorials/p9/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_P9 ${CMAKE_CURRENT_SOURCE_DIR}/p9.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/kde-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/khtml-dll.dll)
 ADD_CS_EXECUTABLE(p9 ${SRC_P9} ALL)
+ADD_DEPENDENCIES(p9 khtml-dll)
diff -Naur kdebindings-4.3.3.orig/csharp/ktexteditor/examples/editor/CMakeLists.txt kdebindings-4.3.3/csharp/ktexteditor/examples/editor/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/ktexteditor/examples/editor/CMakeLists.txt	2008-07-29 13:47:10.000000000 +0200
+++ kdebindings-4.3.3/csharp/ktexteditor/examples/editor/CMakeLists.txt	2009-11-15 11:35:57.000000000 +0100
@@ -5,3 +5,4 @@
 -r:${LIBRARY_OUTPUT_PATH}/ktexteditor-dotnet.dll)
 
 ADD_CS_EXECUTABLE(editor ${SRC_EDITOR} ALL)
+ADD_DEPENDENCIES(editor ktexteditor-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/ktexteditor/examples/kwrite/CMakeLists.txt kdebindings-4.3.3/csharp/ktexteditor/examples/kwrite/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/ktexteditor/examples/kwrite/CMakeLists.txt	2008-07-29 13:47:10.000000000 +0200
+++ kdebindings-4.3.3/csharp/ktexteditor/examples/kwrite/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -5,6 +5,7 @@
 -r:${LIBRARY_OUTPUT_PATH}/ktexteditor-dotnet.dll)
 
 ADD_CS_EXECUTABLE(kwritemain ${SRC_KWRITEMAIN} ALL)
+ADD_DEPENDENCIES(kwritemain ktexteditor-dotnet)
 
 install(FILES kwrite-sharp.desktop DESTINATION ${XDG_APPS_INSTALL_DIR})
 install(FILES kwrite-sharpui.rc DESTINATION ${DATA_INSTALL_DIR}/kwrite-sharp)
diff -Naur kdebindings-4.3.3.orig/csharp/nepomuk/CMakeLists.txt kdebindings-4.3.3/csharp/nepomuk/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/nepomuk/CMakeLists.txt	2009-03-27 15:46:05.000000000 +0100
+++ kdebindings-4.3.3/csharp/nepomuk/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -22,7 +22,7 @@
 
 SET(CS_FLAGS -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll -r:${LIBRARY_OUTPUT_PATH}/soprano.dll -warn:0 -keyfile:${KEYFILE})
 ADD_CS_LIBRARY(nepomuk-dll "${SRC_CS}" ALL)
-ADD_DEPENDENCIES(nepomuk-dll qt-dotnet nepomuk-sharp smokenepomuk)
+ADD_DEPENDENCIES(nepomuk-dll qt-dotnet soprano nepomuk-sharp smokenepomuk)
 
 INSTALL(TARGETS nepomuk-sharp LIBRARY DESTINATION ${LIB_INSTALL_DIR})
 INSTALL_GAC(nepomuk-dll)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/graphicsview/collidingmice/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/graphicsview/collidingmice/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/graphicsview/collidingmice/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/graphicsview/collidingmice/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -4,3 +4,4 @@
 SET(SRC_CS ${CMAKE_CURRENT_SOURCE_DIR}/main.cs ${CMAKE_CURRENT_SOURCE_DIR}/mouse.cs ${CMAKE_CURRENT_BINARY_DIR}/qrc_mouse.cs)
 SET(CS_FLAGS -warn:0 -unsafe -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(collidingmice "${SRC_CS}" ALL)
+ADD_DEPENDENCIES(collidingmice qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/itemviews/simpletreemodel/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/itemviews/simpletreemodel/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/itemviews/simpletreemodel/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/itemviews/simpletreemodel/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -4,3 +4,4 @@
 SET(SRC_CS ${CMAKE_CURRENT_SOURCE_DIR}/main.cs ${CMAKE_CURRENT_SOURCE_DIR}/treemodel.cs ${CMAKE_CURRENT_SOURCE_DIR}/treeitem.cs ${CMAKE_CURRENT_BINARY_DIR}/qrc_simpletreemodel.cs)
 SET(CS_FLAGS -unsafe -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE (simpletreemodel "${SRC_CS}" ALL)
+ADD_DEPENDENCIES(simpletreemodel qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/chat/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/qdbus/chat/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/chat/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/qdbus/chat/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -7,3 +7,4 @@
 SET(SRC_CS_CHAT ${CMAKE_CURRENT_SOURCE_DIR}/chat_adaptor.cs  ${CMAKE_CURRENT_SOURCE_DIR}/chat.cs  ${CMAKE_CURRENT_SOURCE_DIR}/chat_interface.cs  ${CMAKE_CURRENT_BINARY_DIR}/chatmainwindow.cs  ${CMAKE_CURRENT_BINARY_DIR}/chatsetnickname.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(chat "${SRC_CS_CHAT}" ALL)
+ADD_DEPENDENCIES(chat qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/complexpingpong/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/qdbus/complexpingpong/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/complexpingpong/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/qdbus/complexpingpong/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,7 +1,9 @@
 SET(SRC_CS_PING ${CMAKE_CURRENT_SOURCE_DIR}/complexping.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(complexping "${SRC_CS_PING}" ALL)
+ADD_DEPENDENCIES(complexping qt-dotnet)
 
 SET(SRC_CS_PONG ${CMAKE_CURRENT_SOURCE_DIR}/complexpong.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(complexpong "${SRC_CS_PONG}" ALL)
+ADD_DEPENDENCIES(complexpong qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/listnames/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/qdbus/listnames/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/listnames/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/qdbus/listnames/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_CS ${CMAKE_CURRENT_SOURCE_DIR}/listnames.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(listnames "${SRC_CS}" ALL)
+ADD_DEPENDENCIES(listnames qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/pingpong/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/qdbus/pingpong/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/pingpong/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/qdbus/pingpong/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,7 +1,9 @@
 SET(SRC_CS_PING ${CMAKE_CURRENT_SOURCE_DIR}/ping.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(ping "${SRC_CS_PING}" ALL)
+ADD_DEPENDENCIES(ping qt-dotnet)
 
 SET(SRC_CS_PONG ${CMAKE_CURRENT_SOURCE_DIR}/pong.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(pong "${SRC_CS_PONG}" ALL)
+ADD_DEPENDENCIES(pong qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/remotecontrolledcar/car/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/qdbus/remotecontrolledcar/car/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/remotecontrolledcar/car/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/qdbus/remotecontrolledcar/car/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_CS_CAR ${CMAKE_CURRENT_SOURCE_DIR}/car_adaptor.cs  ${CMAKE_CURRENT_SOURCE_DIR}/car.cs  ${CMAKE_CURRENT_SOURCE_DIR}/main.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(car "${SRC_CS_CAR}" ALL)
+ADD_DEPENDENCIES(car qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/remotecontrolledcar/controller/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/qdbus/remotecontrolledcar/controller/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/qdbus/remotecontrolledcar/controller/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/qdbus/remotecontrolledcar/controller/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -4,3 +4,4 @@
 SET(SRC_CS_CONTROLLER ${CMAKE_CURRENT_SOURCE_DIR}/main.cs  ${CMAKE_CURRENT_BINARY_DIR}/ui_controller.cs  ${CMAKE_CURRENT_SOURCE_DIR}/controller.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(controller "${SRC_CS_CONTROLLER}" ALL)
+ADD_DEPENDENCIES(controller qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t1/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t1/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t1/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t1/CMakeLists.txt	2009-11-15 11:36:38.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T1 ${CMAKE_CURRENT_SOURCE_DIR}/t1.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t1 ${SRC_T1} ALL)
+ADD_DEPENDENCIES(t1 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t10/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t10/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t10/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t10/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T10 ${CMAKE_CURRENT_SOURCE_DIR}/t10.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t10 ${SRC_T10} ALL)
+ADD_DEPENDENCIES(t10 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t11/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t11/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t11/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t11/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T11 ${CMAKE_CURRENT_SOURCE_DIR}/cannonfield.cs ${CMAKE_CURRENT_SOURCE_DIR}/lcdrange.cs ${CMAKE_CURRENT_SOURCE_DIR}/main.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t11 "${SRC_T11}" ALL)
+ADD_DEPENDENCIES(t11 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t12/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t12/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t12/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t12/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T12 ${CMAKE_CURRENT_SOURCE_DIR}/cannonfield.cs ${CMAKE_CURRENT_SOURCE_DIR}/lcdrange.cs ${CMAKE_CURRENT_SOURCE_DIR}/main.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t12 "${SRC_T12}" ALL)
+ADD_DEPENDENCIES(t12 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t13/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t13/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t13/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t13/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T13 ${CMAKE_CURRENT_SOURCE_DIR}/cannonfield.cs ${CMAKE_CURRENT_SOURCE_DIR}/gameboard.cs ${CMAKE_CURRENT_SOURCE_DIR}/lcdrange.cs ${CMAKE_CURRENT_SOURCE_DIR}/main.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t13 "${SRC_T13}" ALL)
+ADD_DEPENDENCIES(t13 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t14/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t14/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t14/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t14/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T14 ${CMAKE_CURRENT_SOURCE_DIR}/cannonfield.cs ${CMAKE_CURRENT_SOURCE_DIR}/gameboard.cs ${CMAKE_CURRENT_SOURCE_DIR}/lcdrange.cs ${CMAKE_CURRENT_SOURCE_DIR}/main.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t14 "${SRC_T14}" ALL)
+ADD_DEPENDENCIES(t14 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t2/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t2/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t2/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t2/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T2 ${CMAKE_CURRENT_SOURCE_DIR}/t2.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t2 ${SRC_T2} ALL)
+ADD_DEPENDENCIES(t2 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t3/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t3/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t3/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t3/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T3 ${CMAKE_CURRENT_SOURCE_DIR}/t3.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t3 ${SRC_T3} ALL)
+ADD_DEPENDENCIES(t3 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t4/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t4/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t4/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t4/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T4 ${CMAKE_CURRENT_SOURCE_DIR}/t4.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t4 ${SRC_T4} ALL)
+ADD_DEPENDENCIES(t4 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t5/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t5/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t5/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t5/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T5 ${CMAKE_CURRENT_SOURCE_DIR}/t5.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t5 ${SRC_T5} ALL)
+ADD_DEPENDENCIES(t5 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t6/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t6/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t6/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t6/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T6 ${CMAKE_CURRENT_SOURCE_DIR}/t6.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t6 ${SRC_T6} ALL)
+ADD_DEPENDENCIES(t6 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t7/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t7/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t7/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t7/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T7 ${CMAKE_CURRENT_SOURCE_DIR}/lcdrange.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t7 ${SRC_T7} ALL)
+ADD_DEPENDENCIES(t7 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t8/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t8/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t8/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t8/CMakeLists.txt	2009-11-15 11:37:08.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T8 ${CMAKE_CURRENT_SOURCE_DIR}/t8.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t8 ${SRC_T8} ALL)
+ADD_DEPENDENCIES(t8 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t9/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t9/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/tutorial/t9/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/tutorial/t9/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_T9 ${CMAKE_CURRENT_SOURCE_DIR}/t9.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE(t9 ${SRC_T9} ALL)
+ADD_DEPENDENCIES(t9 qt-dotnet)
diff -Naur kdebindings-4.3.3.orig/csharp/qyoto/examples/widgets/analogclock/CMakeLists.txt kdebindings-4.3.3/csharp/qyoto/examples/widgets/analogclock/CMakeLists.txt
--- kdebindings-4.3.3.orig/csharp/qyoto/examples/widgets/analogclock/CMakeLists.txt	2008-01-05 00:52:37.000000000 +0100
+++ kdebindings-4.3.3/csharp/qyoto/examples/widgets/analogclock/CMakeLists.txt	2009-11-15 11:35:21.000000000 +0100
@@ -1,3 +1,4 @@
 SET(SRC_CS_ANALOGCLOCK ${CMAKE_CURRENT_SOURCE_DIR}/analogclock.cs ${CMAKE_CURRENT_SOURCE_DIR}/main.cs)
 SET(CS_FLAGS -warn:0 -r:${LIBRARY_OUTPUT_PATH}/qt-dotnet.dll)
 ADD_CS_EXECUTABLE (analogclock "${SRC_CS_ANALOGCLOCK}" ALL)
+ADD_DEPENDENCIES(analogclock qt-dotnet)
