--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,6 +32,8 @@
 macro_optional_find_package(KdepimLibs)
 macro_optional_find_package(Phonon)
 macro_optional_find_package(QScintilla)
+macro_optional_find_package(Okular)
+macro_optional_find_package(Plasma)
 
 # Look for Okular
 set( OKULAR_FOUND "no" )
--- a/csharp/CMakeLists.txt
+++ b/csharp/CMakeLists.txt
@@ -106,8 +106,8 @@
     endif(KDEPIMLIBS_FOUND AND Akonadi_FOUND)
 	
 	IF(PLASMA_FOUND)
-		SET(PLASMA_SHARP_ENABLED "yes")
-		ADD_SUBDIRECTORY(plasma)
+		# SET(PLASMA_SHARP_ENABLED "yes")
+		# ADD_SUBDIRECTORY(plasma)
 	ENDIF(PLASMA_FOUND)
 
     ENDIF (GMCS_EXECUTABLE AND GACUTIL_EXECUTABLE AND MONO_CECIL_FOUND)
